packages/esm-commons-lib/src/components/encounter-list/encounter-list.component.tsx

```diff
@@ -75,7 +75,11 @@ export function getObsFromEncounter(encounter, obsConcept, isDate?: Boolean, isT
   const obs = findObs(encounter, obsConcept);
 
   if (isTrueFalseConcept) {
-    return obs ? 'Yes' : 'No';
```

Comment:
This was previously not checking for the concept uuid. This however looks like a possible point of failure for other backends with different a different uuid and might need to be refactored when we come up with a solution for packaging concepts or configurations